frontend/src/store/modules/participant.js

```diff
@@ -4,12 +4,16 @@ export default {
   namespaced: true,
   state: {
     allParticipants: [],
-    error: null
+    error: null,
+    Participant: null
   },
   getters: {
     getParticipants(state) {
       return state.allParticipants;
     },
+    getEmails(state) {
+      return state.allParticipants.filter(list => list.email);
+    },
```

Comment:
@stanislawK @Bosmanfrx if you have a time could you look at these lines? It either renders object that is undefined or doesn't render at all which is why it is undefined (it's also possible that I use wrong the filter() function syntax). Either way I don't get an array of e-mails form the `allParticipants` array of nested json objects as I would want to.